### PR TITLE
Accesibility

### PR DESCRIPTION
**Pull request by Boku52500**
- Page titles - no issues found.
- Image text alternatives - no issues found.
- Text heading - Added H1 and H2 tags to fix the structure.
- Color contrast - Colors are from figma design, exactly the same.
- Resize - no issues found.
- Interaction - Added labels to contact form.
- Moving content - no issues found.
- Multimedia - no issues found/do not apply.
- The basic structure of the page - no issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,10 +29,10 @@
       
       <section id="port-without-header">
          <div class="port-contenido">
-            <h2 id="tit-port" class="title">I’m Ivan Glad to see you!</h2>
```
Comment (christianbiring1) [inline index.html:32]: - [ ]  Kindly make sure you have the hamburger menu in your navbar as it is in the Figma design
Your design :point_down: 
image
Figma design :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -238,8 +238,11 @@ <h3 class="contact-header">Contact me</h3>
       or a project that needs coding. I'd love to help with it </p>
    
     <form action="https://formspree.io/f/mzbwzdgd" class="form-container" method="post" onsubmit="return formSubmit();">
```
Comment (christianbiring1) [inline index.html:243]: - [ ] Kindly make sure you stick to the design 
  Your design :point_down: 
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ivanmvh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights
- [ ] Descriptive PR title, summary
- [ ] All linters checks are green :heavy_check_mark: 

## Required Changes ♻️
- [ ] KIndly make sure you use the same elements spacing:`margin, padding` as in the Figma template
- [ ] Your design :point_down: 

image

- [ ] Figma template :point_down: 

image

- [ ] Kindly consider adding the width of your card.
image

- [ ] Figma design :point_down: 
image

- [ ] KIndly make sure your element are well positioned, `You can use display flex or grid`
Your design :point_down: 

image

- [ ] Figma design :point_down: 
image

- [ ] Your design :point_down: 
image

- [ ] Figma template :point_down: 
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]  Write some good `animations` and/or `transition`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Boku52500** [comment]
- Updated the design of Portfolio according to figma.
- Page titles - no issues found.
- Image text alternatives - no issues found.
- Text heading - Added H1 and H2 tags to fix the structure.
- Color contrast - Colors are from figma design, exactly the same.
- Resize - no issues found.
- Interaction - Added labels to contact form.
- Moving content - no issues found.
- Multimedia - no issues found/do not apply.
- The basic structure of the page - no issues found.

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi team👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

## Optional suggestions:
- [ ] The images of your website are not showing in the live demo, please consider fixing this issue 💯 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**